packages/vite/src/node/plugins/manifest.ts

```diff
@@ -67,6 +68,7 @@ export function manifestPlugin(config: ResolvedConfig): Plugin {
         }
         if (chunk.isEntry) {
           manifestChunk.isEntry = true
+          manifestChunk.name = chunk.name
```

Comment:
Is there any reason to only populate the `name` for entry chunks?

Reply:
👍 I think it's a good idea. I just pushed a commit to always add `chunk.name`.